WD-Firmware/board/nexys_a7/bsp/env/freedom-e300-hifive1/openocd.cfg

```diff
@@ -0,0 +1,34 @@
+adapter_khz     10000
```

Comment:
remove/rename the folder freedom-e300-hifive1

Reply:
[will be done after new main]

Reply:
Done by Ofer